packages/components/src/components/hds/alert/index.ts

```diff
@@ -178,8 +155,20 @@ export default class HdsAlert extends Component<HdsAlertSignature> {
     const actions = element.querySelectorAll(
       `${CONTENT_ELEMENT_SELECTOR} button, ${CONTENT_ELEMENT_SELECTOR} a`
     );
-    if (actions.length) {
+
+    const isRealAlert: boolean =
+      this.color === 'warning' ||
+      this.color === 'critical' ||
+      this.color === 'success';
+
+    /*
+      Q: If the alert contains actions but is not a "real alert", should it be an alertdialog? 
+      (Currently assuming it is not but don't know if that's correct)
```

Comment:
@MelSumner I wasn't sure about what role Alerts should have if they are used for info or promo purposes but contain actions.

Reply:
They should not have any role if they are used only for informational or promotional purposes. 

The difference between `alert` and `alertdialog` (as we've chosen to define it) is that if the component contains an interactive element, then it should be `alertdialog`. An element with the role of `alert` should contain string content (when flattened).